pkgs/applications/misc/houdini/runtime.nix

```diff
@@ -50,11 +58,13 @@ stdenv.mkDerivation rec {
                       --no-root-check \
                       --accept-EULA \
                       $out
-    sed -i "s|/usr/lib/sesi|${license_dir}|g" $out/houdini/Licensing.opt
+    echo -e "localValidatorDir = ${license_dir}\nlicensingMode = localValidator" > $out/houdini/Licensing.opt
```

Comment:
I don't think it's related. Making the license checker happy was the main chore in writing this package in the first place. I stopped fiddling with things as soon as it started working on my system, @antonxy is just removing a redundant hack that I left in and changing a config option that should have been changed. (I wonder why it worked without that change on my box though?)